packages/graphql/package.json

```diff
@@ -32,5 +47,20 @@
     "lint": "eslint src/ test/ --report-unused-disable-directives --max-warnings=0",
     "lint:fix": "eslint . --report-unused-disable-directives --fix"
   },
-  "private": true
+  "files": [
+    "lib/*.tsp",
+    "dist/**",
+    "!dist/test/**"
+  ],
+  "peerDependencies": {
+    "@typespec/compiler": "workspace:~"
+  },
+  "devDependencies": {
+    "@types/node": "~22.7.5",
+    "@typespec/compiler": "workspace:~",
```

Comment:
why do we need to add this to devDependencies?

Reply:
We cannot run tests without the compiler. The same is used in other TypeSpec packages.